### ISSUE TITLE
NTR: 3-hydroxykynurenine high in urine (Lab: chemistry: amino compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8509]
New term request 3-hydroxykynurenine high in urine (segal_030502192518):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000265
- label: Elevated urinary 3-hydroxykynurenine level
- parents: [Abnormal urinary non-proteinogenic amino acid level (HP:0034027)](https://hpo.jax.org/app/browse/term/HP:0034027)
- pmid: PMID:17334708
- def: The amount of 3-hydroxykynurenine in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8509
- synonyms:
